### PR TITLE
feat: adjust build for PlatformApps to upload Folders

### PR DESCRIPTION
**Pull request by julianiff**
-> New Build for Platform apps that build a complete folder and upload the folder content to the Marketplace.

### CODE REVIEW

File: packages/cli/tests/utils/compiler.spec.ts [hunk 1]
```diff
@@ -72,10 +61,9 @@ describe('Compiler utils', () => {
                 outputName: outputNameTest,
             })) as unknown as { app: { output: { fileName: string }[] }; settings: { output: { fileName: string }[] } };
 
-            expect(createHash).toHaveBeenCalledWith('sha256');
-            expect(result.app.output[0].fileName).toBe(`${outputNameTest}.${testHash}.js`);
-            expect(result.app.output[1].fileName).toBe(`${outputNameTest}.${testHash}.css`);
-            expect(result.app.output[2].fileName).toBe(`${outputNameTest}.${testHash}.html`);
+            expect(result.app.output[0].fileName).toBe('assets/index-sM5bF_1u.js');
+            expect(result.app.output[1].fileName).toBe('assets/index-B-x5KKng.css');
```
Comment (imoutaharik) [inline packages/cli/tests/utils/compiler.spec.ts:65]: 🤔 Where does the code in the filename comes from? I'm wondering why it's not `index.js` or `index.css` like the last 2 that have a proper name. (index.html, settings.js)

Comment (julianiff) [reply packages/cli/tests/utils/compiler.spec.ts:65]: It is after the build, vite puts a unique identifer on the js/css files to break cache.

Comment (imoutaharik) [reply packages/cli/tests/utils/compiler.spec.ts:65]: Got it! Thanks.

File: packages/cli/src/utils/compiler.ts [hunk 2]
```diff
@@ -94,33 +92,8 @@ export const compileBlock = async ({ projectPath, entryFile, outputName }: Compi
 };
 
 export const compilePlatformApp = async ({ outputName, entryFile, projectPath = '' }: CompilerOptions) => {
-    const getHash = (text) => createHash('sha256').update(text).digest('hex');
     const appBridgeVersion = getAppBridgeVersion(projectPath);
 
-    const htmlHashPlugin: PluginOption = {
```
Comment (ragi96) [inline packages/cli/src/utils/compiler.ts:100]: Lets split the compile in a follow up so it doesn't get noisy is something gets changed in here, and each team can fully own their compile 👍

File: .changeset/pink-houses-pump.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": patch
+---
+
+feat: adjust build to be a folder
```
Comment (ragi96) [inline .changeset/pink-houses-pump.md:5]:
```suggestion
feat(PlattformApp): adjust build to be a folder
```

File: .changeset/pink-houses-pump.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": patch
+---
+
+feat(PlattformApp): adjust build to be a folder
```
Comment (julianiff) [inline .changeset/pink-houses-pump.md:5]:
```suggestion
feat(PlatformApp): adjust build to be a folder
```

**Review by ragi96 — APPROVED** [review]

**Review by mike85 — APPROVED** [review]
LGTM

**Pull request merged**